mistletoe/contrib/mathjax.py

```diff
@@ -6,10 +6,17 @@
 from mistletoe.latex_renderer import LaTeXRenderer
 
 class MathJaxRenderer(HTMLRenderer, LaTeXRenderer):
-    """
-    MRO will first look for render functions under HTMLRenderer,
-    then LaTeXRenderer.
-    """
+    def __init__(self, **kwargs):
+        """
+        MRO will first look for render functions under HTMLRenderer,
```

Comment:
Maybe "MRO" deserves an explanation?

Reply:
Yeah, maybe, it is <https://docs.python.org/3/library/stdtypes.html#class.__mro__>. But I didn't want to dig any deeper for now. :P

Reply:
How about changing "MRO" to "Python", then? It would make the comment understandable without digging into the technicalities.

Reply:
(It is off-topic for this PR, for sure. But still could make sense, as the line is touched.)

Reply:
You;re right, we are moving it anyway. It is rather an inner workings note, explaining what Python does when there are multiple ancestors, so I would remove it altogether, what do you say?

Reply:
Yes, sounds good!

Reply:
Done. :)